jwt/audience.go

```diff
@@ -1,79 +1,33 @@
 package jwt
 
 import (
-	"errors"
-	"fmt"
-	"net"
-	"net/url"
-	"strings"
 )
 
-// Audience must be the base URL from the request including protocol, domain,
-// and port if applicable for the domains you specify in your IAP proxy. For
-// example, https://example.com or https://foo.example.com:port.
-type Audience url.URL
+// From the IAP docs at https://cloud.google.com/iap/docs/signed-headers-howto:
+// Audience must be a string with the following values:
+// * App Engine: /projects/PROJECT_NUMBER/apps/PROJECT_ID
+// * Compute Engine and Container Engine: /projects/PROJECT_NUMBER/global/backendServices/SERVICE_ID
+type Audience string
 
-// NewAudience returns an Audience from a URL.
-func NewAudience(u *url.URL) *Audience {
+// NewAudience returns an Audience from a string.
+func NewAudience(u *string) *Audience {
```

Comment:
Probably doesn't make sense to create a nil-able audience.
Do you see any problem with using `string` directly?